images/vm-image-driver.bb

```diff
@@ -17,6 +17,12 @@ IMAGE_FEATURES += " \
     benchmark \
     "
 
+IMAGE_INSTALL += " \
+    trace-cmd \
+    sel4-trace-support \
+    sel4-collect-traces \
+    "
```

Comment:
Shouldn't the closing quotation mark be at the same indent level as `IMAGE_INSTALL`? I know Yocto's devtool generates it like that but to me it would be more aesthetical if you removed the spaces here?

Reply:
I think that for Yocto as it just Python script it does not matter. As for me I would prefer to keep it on the same indent level, but in this case I need also fix all other indents in this file and probably in all files here as they use such weird indent, but this is not a purpose of this one PR... But again if you suggest so, I probably could fix them all =)

Reply:
OK, I don't want to be blocker in this regard. This is fine.